openapi/v2/core-metadata.yaml

```diff
@@ -552,6 +562,147 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/DeviceService'
+    ProvisionWatcher:
+      description: "A ProvisionWatcher defines the filtering criteria for device auto discovery."
+      type: object
+      properties:
+        apiVersion:
+          type: string
+        id:
+          type: string
+          format: uuid
+          description: Uniquely identifies this request. For implementation, recommend this value be generated by the type's constructor
+        name:
+          type: string
+          description: Non-database identifier (must be unique)
+        created:
+          type: integer
+          description: Created is a timestamp indicating when the entity was created.
+        modified:
+          type: integer
+          description: Modified is a timestamp indicating when the entity was last modified.
+        labels:
+          type: array
+          description: Other labels applied to the device to help with searching
+          items:
+            type: string
+        identifiers:
+          type: object
+          description: Set of key value pairs that identify property (MAC, HTTP,...) and value to watch for (00-05-1B-A1-99-99, 10.0.0.1,...)
+          additionalProperties:
+            type: string
+        blockingIdentifiers:
+          type: object
+          description: Set of key-values pairs that identify devices which will not be added despite matching on identifiers
+          additionalProperties:
+            type: array
+            items:
+              type: string
+        profile:
```

Comment:
Other DTOs use `deviceProfile`. Do same here for consistency. Here and below

Reply:
Device use `serviceName` and `profileName`, I'll make the same change for consistency